packages/at_onboarding_cli/lib/src/activate_cli/activate_cli.dart

```diff
@@ -41,7 +43,10 @@ Future<void> main(List<String> arguments) async {
 
   stdout.writeln('[Information] Root server is ${argResults['rootServer']}');
 
-  var downloadPath = '${Directory.current.path}/keys';
+  // var downloadPath = '${Directory.current.path}/keys';
+  var homeDirectory = getHomeDirectory();
+  var downloadPath = "$homeDirectory/.atsign/keys";
```

Comment:
Do you need to use platform-specific path separators? e.g. \ for Windows etc

Reply:
We use the same code in attalk on windows. I really need to put those into a package..

Reply:
Updated ready for v1.2.5 release to pub.dev